pkg/operators/machine-api-operator.go

```diff
@@ -164,3 +166,72 @@ var _ = Describe("[Feature:Operators] Machine API cluster operator status should
 		Expect(framework.IsStatusAvailable(client, "machine-api")).To(BeTrue())
 	})
 })
+
+var _ = Describe("[Feature:Operators] When cluster-wide proxy is configured, Machine API cluster operator should ", func() {
+	defer GinkgoRecover()
```

Comment:
Shouldn't need GinkgoRecover unless you have an go routines in your tests, which I can't see

Reply:
good know, I pulled that out.